feeds/public.txt

```diff
@@ -84,5 +84,6 @@ fluid-framework
 @fluid-experimental/property-dds
 @fluid-experimental/property-common
 @fluid-experimental/property-changeset
+@fluidframework/eslint-config-fluid
```

Comment:
We probably don't want this published publicly with the version that it acquires as part of the client release group, right? In fact, probably not published at all anywhere...? Not even the build feed.